server/src/main/java/com/google/coffeehouse/common/StorageHandler.java

```diff
@@ -102,61 +224,110 @@ public static String getClubQuery(DatabaseClient dbClient, String clubId) {
       try (ResultSet resultSet = dbClient.singleUse().executeQuery(statement)) {
         while (resultSet.next()) {
           bookId = resultSet.getString("bookId");
+          name = resultSet.getString("name");
           description = resultSet.getString("description");
           ownerId = resultSet.getString("ownerId");
-          name = resultSet.getString("name");
         }
-        clubInfo = String.format(
-                          "Club ID: %s || Book ID: %s || Description: %s || Name: %s || Owner ID: %s\n",
-                          clubId, bookId, description, name, ownerId);
       }
+      Book book = getBookQuery(dbClient, bookId);
+      Club.Builder clubBuilder = Club.newBuilder(name, book);
+      club = clubBuilder
+                        .setDescription(description)
+                        .build();
     } else if (resultCount > 1) {
-      clubInfo = ERROR_MORE_THAN_ONE_CLUB;
+      throw new IllegalArgumentException(ERROR_MORE_THAN_ONE_CLUB);
     } else {
-      clubInfo = CLUB_DOES_NOT_EXIST;
+      throw new IllegalArgumentException(CLUB_DOES_NOT_EXIST);
     }
-    return clubInfo;
+    return club;
   }
 
   /**
-  * Returns a string containing information about a book queried from the database.
-  * This method formats a string with a book's ID, author, ISBN, and title.
-  * @param  dbClient  the database client
-  * @param  bookId    the book ID string used to query and get a book's information
-  * @return bookInfo  the formatted string containing the book information
+  * Creates and returns a list of {@link Persons}s that are a member of a club.
+  * This method builds a {@link Person} for each person who is a member of the club
+  * specified by club ID. Each {@link Person} is added to a list that gets returned.
+  * @param  dbClient    the database client
+  * @param  clubId      the club ID string used to query
+  * @return persons     the list of Person objects
   */
-  public static String getBookQuery(DatabaseClient dbClient, String bookId) {
-    String bookInfo = "";
-    String author = "";
-    String isbn = "";
-    String title = "";
-    long resultCount = StorageHandlerHelper.getBookCountQuery(dbClient, bookId);
+  public static List<Person> getListOfMembersQuery(DatabaseClient dbClient, String clubId) {
+    List<Person> persons = new ArrayList<>();
+    long resultCount = StorageHandlerHelper.getMemberCountQuery(dbClient, clubId);
     Statement statement = 
         Statement.newBuilder(
-                "SELECT bookId, author, isbn, title "
-                  + "FROM Books "
-                  + "WHERE bookId = @bookId")
-            .bind("bookId")
-            .to(bookId)
+                "SELECT userId, clubId, membershipType "
+                  + "FROM Memberships "
+                  + "WHERE clubId = @clubId AND membershipType = " + Membership.MEMBER)
```

Comment:
I see no reason to not count the owner as a member of the club, otherwise it could be confusing.